js/main.js

```diff
@@ -0,0 +1,26 @@
+//Функция возврата целого числа, взято из MDN
+
+function getWholeNumber(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+getWholeNumber(78, 99);
+
+// функция возврата положительного числа с плаваующей точкой
+let fractionalNumber;
+
+function getNumber(min, max, round) {
+  if(min >= 0 && max >= 0) {
```

Comment:
Форматировать лучше так
```suggestion
  if (min >= 0 && max >= 0) {
```